setup/setup.iss

```diff
@@ -266,15 +258,18 @@ begin
     else if RegQueryStringValue(HKEY_LOCAL_MACHINE, 'SOFTWARE\GOG.com\GOGREDFACTION', 'PATH', Result) then
         // GOG
         Result := Result + 'RF.exe'
+    else if RegQueryStringValue(HKEY_LOCAL_MACHINE, 'SOFTWARE\GOG.com\Games\1207660623', 'PATH', Result) then
+        // GR GOG
+        Result := Result + '\RF.exe'
     else
         // Fallback
-        Result := 'C:\games\RedFaction\RF.exe';
+        Result := ExpandConstant('{sd}\games\RedFaction\RF.exe');

```

Comment:
Have you ever seen Windows being on a different drive? Do you know if RF setup program also uses this default? I couldn't find any information about changing system drive in windows. 
Unless there is a reason for this change I would prefer to keep it the old way at is was simpler to understand and consistent with the launcher.

Reply:
I added that to cover the possible scenario that someone may be running their OS from an external drive or the D:\ partition (I've installed Windows to it before on a different computer and ran different games under it). That's it really.